File: two_elevators/two.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int cn;
    scanf("%d", &cn);
    for (int i = 0; i < cn; i++)
    {
        int a, b, c;
        scanf("%d%d%d", &a, &b, &c);
        int first = abs(a - 1);
        int second = 0;
        if (c == 1)
        {
            second = (abs(b - c));
        }

        else
        {
            second = abs(b - c) + abs(c - 1);
        }

        if (first < second)
        {
            printf("%d\n", 1);
        }

        else if (second < first)
        {
            printf("%d\n", 2);
        }

        else
        {
            printf("%d\n", 3);
        }
    }
}

File: two_elevators/two.py
def main():
    cn = int(input())
    for _ in range(cn):
        a, b, c = map(int, input().split())
        first = abs(a - 1)
        second = abs(b - c) + abs(c - 1)
        if first < second:
            print(1)
        elif second < first:
            print(2)
        else:
            print(3)
    
    
if __name__ == "__main__":
    main()
